pkg/driver/driver.go

```diff
@@ -478,7 +492,7 @@ func (d Driver) ControllerUnpublishVolume(ctx context.Context, req *csi.Controll
 	vol, err := d.Storage.GetByID(ctx, req.VolumeId)
 	if err != nil {
 		return nil, status.Errorf(codes.Internal,
-			"ControllerUnpublishVolume failed for %s: %v", req.GetVolumeId(), err)
+			"ControllerUnpublishVolume failed for %s: %v",req.GetVolumeId(), err)
```

Comment:
This is just one example, but when I open and close files, it looks like you did not use gofmt (or and IDE that formats Go code correctly). Please do so, otherwise this produces diffs on follow up commits. `gofmt` is the canonical way.